### ISSUE TITLE
Burp Enterprise Parser - Empty CWE Not Supported

### ISSUE THREAD

**Issue by matamorphosis** [DefectDojo/django-DefectDojo#6498]
Apologies for not slacking you first, but as I spent several hours debugging this myself, I felt this route more prudent.

**Be informative**
When setting up our SAST/DAST CI pipeline jobs, I have built a scheduled job that pulls the latest BurpSuite Enterprise scan report for a list of sites that are being scanned, and forwards them to the relevant engagement/test ID in DefectDojo. I identified that some reports would successfully be imported while others weren't, when I tried to manually reproduce the issue, by doing a re-import of the problematic report into DefectDojo. I was met with the following error:

```
An exception error occurred during the report import:invalid literal for int() with base 10: ''
```

I compared a successful report to an unsuccessful report, and noted that the successful reports had CWEs for each finding, and the absence of a CWE from a finding would cause an exception. This was only noted to be an issue with Info-level findings. All Low-High findings in the report had a CWE. This being said I would suggest the parser be updated to not require a CWE from each finding. For the meanwhile I have worked around this issue by only forwarding Low-High severity findings to DefectDojo.

By following the flow of how reports are ingested into DefectDojo, I was able to determine that the issue exists in the Burp Enterprise [parser.py file, on line 178](https://github.com/DefectDojo/django-DefectDojo/blob/master/dojo/tools/burp_enterprise/parser.py#L178)

I was able to determine the location of this by looking at the application server logs.

**Bug description**
A clear and concise description of what the bug is. For errors include at least the exact error message you are seeing (including traceback).

**Steps to reproduce**
1. You would need a BSE scan report with info findings without CWEs
2. Go to an engagement and selection the "Import Scan Results" option
3. Upload the problematic report.

**Expected behavior**
The application will not accept the report due to it not being able to identify a CWE for all findings.

**Logs** 
Use `docker-compose logs` to see the error.

**Comment by StefanFl** [comment]
Hi @matamorphosis, thank you for this bug report. Can you provide a sample report to reproduce the bug? It would help building a unit test case for the fix.

**Comment by matamorphosis** [comment]
@StefanFl the best I can do is provide a real report that is heavily-redacted. I have confirmed this reproduces the error, but is an altered version of the original report, that was how I discovered the issue.
[BSE_results.html.zip](https://github.com/DefectDojo/django-DefectDojo/files/9034653/BSE_results.html.zip)

**Comment by StefanFl** [comment]
Merged for release 1.12
